spec/controllers/photos_controller_spec.rb

```diff
@@ -54,6 +54,12 @@ def valid_session
       assigns(:type).should eq "garden"
```

Comment:
Controller and view specs are both covered by feature specs, which does 
the full webserver roundtrip.  Model tests are separate.

And yes, we should be moving stuff out of controllers/views into 
features where possible, so they eventually all disappear.

On 26/02/2015 8:22 am, Mackenzie wrote:
>
> In spec/controllers/photos_controller_spec.rb 
> <https://github.com/Growstuff/growstuff/pull/712#discussion_r25380198>:
>
> > @@ -54,6 +54,12 @@ def valid_session
> >        assigns(:type).should eq "garden"
>
> Controller and view specs are BOTH deprecated? Well it doesn't make 
> sense to me that this get moved to...model?... spec without also 
> moving all the rest of it, because splitting up a test that's done 
> almost identically (but with different input) several times seems 
> really disorganized.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Growstuff/growstuff/pull/712/files#r25380198>.
>

-- 
Alex "Skud" Bayley
skud@growstuff.org
http://growstuff.org/